apis/kueue/v1beta2/clusterqueue_types.go

```diff
@@ -191,7 +199,8 @@ type ResourceGroup struct {
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=64
 	// +listType=atomic
-	CoveredResources []corev1.ResourceName `json:"coveredResources"`
+	// +required
+	CoveredResources []corev1.ResourceName `json:"coveredResources,omitempty"`
```

Comment:
omitempty looks weird for required values, especially when we have `MinItems=1`? Did you add it because the linter complained? TBH I would rather add a comment to disable linter (same below for the Flavors field).

Reply:
Yes the linter requires it.